lib/api/v3/queries/query_representer.rb

```diff
@@ -260,7 +260,13 @@ class QueryRepresenter < ::API::Decorators::Single
                  setter: ->(fragment:, **) {
                    next unless represented.new_record?
 
+                   # Check which of the work package IDs exist
+                   positions = Hash(fragment)
+                   existent_wps = WorkPackage.where(id: positions.keys).pluck(:id).to_set
+
                    Hash(fragment).each do |wp_id, position|
+                     next unless existent_wps.include?(wp_id.to_i)
```

Comment:
I thinks this is the wrong place to add the functionality. It should rather be placed in the query services (or the base service) as it is business logic IMO to disregard inexistent ids.

Reply:
Good point, I addressed this